### PR TITLE
Enable Front End Edit Venue Link in CE

### PR DESCRIPTION
**Pull request by jesseeproductions**
_Ref:_ [C#63099 ](https://central.tri.be/issues/63099 )

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Must use &quot;self::&quot; for local static member reference | [Main.php:1637](https://github.com/moderntribe/the-events-calendar/blob/2f219e901eef21af3275735da5df598d7612c3fe/src/Tribe/Main.php#L1637)

> via jenkins/codesniffer-phpcs

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -1631,9 +1631,14 @@ public function displayEventVenueDropdown( $post_id ) {
 					Tribe__Events__Linked_Posts::instance()->saved_linked_post_dropdown( Tribe__Events__Venue::POSTTYPE, $venue_id );
 					$venue_pto = get_post_type_object( self::VENUE_POST_TYPE );
 					if ( current_user_can( $venue_pto->cap->edit_posts ) ) {
+						//Use Admin Link Unless on Community Events Editor then use Front End Link to Edit
+						$edit_link = admin_url( sprintf( 'post.php?action=edit&post=%s', $venue_id ) );
+						if ( tribe_is_community_edit_event_page() ) {
```
Comment (borkweb) [inline src/Tribe/Main.php]: I find it so weird that this method is in the-events-calendar and not events-community. Just a comment on life in general - not about the state of the code in this PR :)

File: src/Tribe/Main.php [hunk 2]
```diff
@@ -1631,9 +1631,14 @@ public function displayEventVenueDropdown( $post_id ) {
...
+							$edit_link = Tribe__Events__Community__Main::instance()->getUrl( 'edit', $venue_id, null, self::VENUE_POST_TYPE );
+						}
 						?>
-						<div class="edit-venue-link" <?php if ( empty( $venue_id ) ) { ?>style="display:none;"<?php } ?>>
-							<a data-admin-url="<?php echo esc_url( admin_url( 'post.php?action=edit&post=' ) ); ?>" href="<?php echo esc_url( admin_url( sprintf( 'post.php?action=edit&post=%s', $venue_id ) ) ); ?>" target="_blank">
+						<div class="edit-venue-link " <?php if ( empty( $venue_id ) ) { ?>style="display:none;"<?php } ?>>
```
Comment (borkweb) [inline src/Tribe/Main.php]: An extra space snuck into the `class` attribute.

**Comment by bordoni** [comment]
👍

**Pull request merged**